content/docs/user-guide/components/reference/terrain/world-renderer.md

```diff
@@ -22,3 +36,27 @@ You must add this component to the *Level* entity, the parent of all entities in
 ## Properties
 
 ![Terrain World Renderer component interface.](/images/user-guide/components/reference/terrain/terrain-world-renderer-A.png)
+
+| Property | Description | Values | Default |
+| - | - | - | - |
+| **Mesh Configuration** |
+| Mesh render distance | The distance from the camera that terrain meshes will render. | 1.0 to 100000.0 | 4096.0 |
+| First LOD distance | The distance from the camera that the first Lod renders to. Subsequent LODs will be at double the distance from the previous LOD. | 1.0 to 10000.0 | 128.0 |
+| Continuous LOD (CLOD) | Enables the use of continuous level of detail, which smoothly blends geometry between terrain lods. | Boolean | true |
+| CLOD Distance | Distance in meters over which the first lod will blend into the next lod. Subsequent lod blend distances will double with each lod for a consistent visual appearance. | 0.0 to 1000.0 | 16.0
+| |
+| **Detail Material Configuration** |
+| Height based texture blending | When turned on, detail materials will use the height texture to aid with blending. | Boolean | 0.0 |
```

Comment:
Default is "False", not "0.0". (Side note - should we change this to default to True?)

Reply:
We probably should set this to true, but it's false right now. I'll update the wording.